lib/bitbucket_rest_api/repos/forks.rb

```diff
@@ -0,0 +1,52 @@
+# encoding: utf-8
+
+module BitBucket
+  class Repos::Forks < API
+
+    REQUIRED_KEY_PARAM_NAMES = %w[ name ].freeze
+    DEFAULT_REPO_OPTIONS = {
+        "website"         => "",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.